webview-ui/src/components/chat/ChatView.tsx

```diff
@@ -2,7 +2,6 @@ import { VSCodeButton, VSCodeLink } from "@vscode/webview-ui-toolkit/react"
 import debounce from "debounce"
 import { useCallback, useEffect, useMemo, useRef, useState } from "react"
 import { useDeepCompareEffect, useEvent, useMount } from "react-use"
-import { Virtuoso, type VirtuosoHandle } from "react-virtuoso"
```

Comment:
why you remove Virtuoso ?, this will make memory go up

Reply:
The PR description explains the reasoning and also mentions:
> No observed performance loss, no observed memory rise.

I want to spend some time profiling the memory though, just for my own edification.